From b6dcae8ec4334e2436cfa19a8052c3473f57a586 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 4 Mar 2017 15:02:43 +0100
Subject: [PATCH 1/2] src/Makefile.am: don't use VPATH

VPATH is a mechanism used to allow out-of-tree build, not to use source
code from a different directory. For this purpose, one should simply use
the "subdir-objects" AUTOMAKE_OPTIONS, and reference the source code of
the sub-directory in the <prog>_SOURCES variable.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Upstream-status: https://github.com/irontec/sngrep/pull/176
---
 src/Makefile.am | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 931ee06..961f4a0 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,5 +1,5 @@
+AUTOMAKE_OPTIONS=subdir-objects
 bin_PROGRAMS=sngrep
-VPATH=curses
 sngrep_SOURCES=capture.c
 if USE_EEP
 sngrep_SOURCES+=capture_eep.c
@@ -12,8 +12,8 @@ sngrep_SOURCES+=capture_openssl.c
 endif
 sngrep_SOURCES+=address.c packet.c sip.c sip_call.c sip_msg.c sip_attr.c main.c
 sngrep_SOURCES+=option.c group.c filter.c keybinding.c media.c setting.c rtp.c
-sngrep_SOURCES+=util.c hash.c vector.c ui_panel.c scrollbar.c
-sngrep_SOURCES+=ui_manager.c ui_call_list.c ui_call_flow.c ui_call_raw.c
-sngrep_SOURCES+=ui_stats.c ui_filter.c ui_save.c ui_msg_diff.c
-sngrep_SOURCES+=ui_column_select.c ui_settings.c
+sngrep_SOURCES+=util.c hash.c vector.c curses/ui_panel.c curses/scrollbar.c
+sngrep_SOURCES+=curses/ui_manager.c curses/ui_call_list.c curses/ui_call_flow.c curses/ui_call_raw.c
+sngrep_SOURCES+=curses/ui_stats.c curses/ui_filter.c curses/ui_save.c curses/ui_msg_diff.c
+sngrep_SOURCES+=curses/ui_column_select.c curses/ui_settings.c
 
-- 
2.7.4

